README.md

```diff
@@ -260,138 +260,168 @@ We gladly credit all contributors.
-      improving the value of the company as a whole).
+      stock, since they prefer employees hold their stock and work to make it more valuable by
+      improving the value of the company as a whole.
     - But former employees and other shareholders often have difficulty initiating secondary
       transactions as well.
       Private buyers may ask for the company’s internal financials in order to estimate the
       current and future value of its stock;
       the company may not wish to share this confidential information.
       Companies
-      [also will want to consider](https://www.secondariesinvestor.com/new-normal-secondaries-409a-valuation-process/)
-      whether sales could influence their [409A valuation](#409a-valuations).
+      [also consider](https://www.secondariesinvestor.com/new-normal-secondaries-409a-valuation-process/)
+      whether sales could influence their [409A valuation](#409a-valuations). - 🚧 This link
+      requires registration to access.
+      Are we okay with that?
+      Also, do they “consider” or do they “worry”?
     - Secondary sales are also an administrative and legal burden that may not make it to the
       top of the list of priorities for busy startup CEOs and CFOs.
     - In late-stage funding rounds, companies frequently do secondary offerings where employees
       may be eligible to sell a portion of their stock.
-  - **Secondary buyers:**
-    Thinking on all this
+  - **Secondary buyers**: Thinking on all this
     [has evolved](http://www.industryventures.com/2014/12/02/employee-liquidity-good-for-private-companies/)
-    [in recent years](https://techcrunch.com/2015/10/14/selling-private-company-shares-2-0/);
-    some companies do see value in offering (mostly limited) opportunities for sale.
+    [in recent years](https://techcrunch.com/2015/10/14/selling-private-company-shares-2-0/).
+    Some companies do see value in offering (mostly limited) opportunities for sale.
     [SharesPost](http://sharespost.com/), [Equidate](https://www.equidateinc.com/), and
     [EquityZen](https://equityzen.com/) have sought to establish a market around secondary sales,
     particularly for well-known pre-IPO companies.
-    A few other secondary firms have emerged that have interest in certain purchases,
-    especially for larger sales from founders, early employees, or executives (parties
-    generally in possession of larger shares).
-    A company can work with the firm to facilitate multiple transactions.
-    These firms include [137 Ventures](http://137ventures.com/),
-    [ESO Fund](https://employeestockoptions.com/),
-    [Akkadian Ventures](https://www.akkadianventures.com/),
-    [Industry Ventures](http://www.industryventures.com/), [Atlas Peak](http://www.atlaspeakcap.com/),
-    and [Founders Circle](http://www.founderscircle.com/).
+    \- 🚧 Briefly share what these companies see as valuable in offering opportunities for
```

Comment:
Just FYI to watch out for, this is a error that can crop up with Flowmark if indentation isn't quite right. If it happens delete the `\` and make sure it's indented correctly. Adding to track. https://github.com/jlevy/atom-flowmark/issues/12